packages/common/src/ceramic-api.ts

```diff
@@ -60,29 +60,34 @@ export interface CeramicSigner extends CeramicCommon {
 export interface AdminApi {
   /**
    * List indexed model streams
+   *
+   * @param actingDid - did that performs the operation
    */
-  getIndexedModels(): Promise<Array<StreamID>>
+  getIndexedModels(actingDid: DID | string): Promise<Array<StreamID>>
```

Comment:
will this ever be a DID instance?  Isn't it always a string, at least for now?  So can we tighten up the interface and make it explicitly always the DID string?

Reply:
It's a did on the client side (see `RemoteAdminApi` :).

Reply:
Ah right, and I do see now that in the actual Local/RemoteAdminApi classes you use just the proper specific type that is expected in each context.  Not sure exactly how Typescript's compiler handles that kind of type specialization in interface implementations, but hopefully it does what we want 🙂